docs/docs/end-users/Configuration.fsx

```diff
@@ -0,0 +1,171 @@
+<p><i class="bi bi-check-circle-fill green-recommendation fs-4 align-middle" ></i> <strong>Safe to change:</strong> Settings that aren't attached to any guidelines. Depending on your team or your own preferences, feel free to change these as it's been agreed on the codebase, however, you can always use it's defaults.</p>
+<p><i class="bi bi-exclamation-circle-fill orange-recommendation fs-4 align-middle"></i> <strong>Not recommended:</strong> Settings that don't follow any guidelines.</p>
+<p><i class="bi bi-x-circle-fill red-recommendation fs-4 align-middle"></i> <strong>Do not use:</strong> Settings you should never enable. They might lead to incomplete results.</p>
+<p><img class="gresearch-recommendation align-middle" src="{{root}}/gresearch.svg" alt="G-Research logo"/> <strong>G-Research:</strong> G-Research styling guide. If you use one of these, for consistency reasons you should use all of them.</p>
+*)
+
+(**
+## Auxiliary settings
+#### <i class="bi bi-check-circle-fill green-recommendation me-1 fs-5" data-bs-toggle="tooltip" data-bs-custom-class="green-tooltip" data-bs-title="This setting is good to use" ></i>indent_size
+` indent_size` has to be between 1 and 10.
+
+This preference sets the indentation.  
+The common values are 2 and 4. 
+The same indentation is ensured to be consistent in a source file.
+Default = 4.
+
+`defaultConfig`
+*)
+
+formatCode
+    """ 
+    let inline selectRandom (f: _ []) =
+        let r = random 1.0
+    
+        let rec find =
+            function
+            | 0 -> fst f.[0]
+            | n when r < snd f.[n] -> fst f.[n]
+            | n -> find (n - 1)
+    
+        find <| f.Length - 1
+    """
+    { FormatConfig.Default with IndentSize = 2 }
+(*** include-it ***)
+
+(**
+## Maximum width constraints
+#### <i class="bi bi-check-circle-fill orange-recommendation me-1" data-bs-toggle="tooltip" data-bs-custom-class="orange-tooltip" data-bs-title="This setting is not recommended"></i> indent_size
+` indent_size` has to be between 1 and 10.
+
+This preference sets the indentation
+The common values are 2 and 4. 
+The same indentation is ensured to be consistent in a source file.
+Default = 4.
+
+`defaultConfig`
+*)
+
+formatCode
+    """ 
+    let inline selectRandom (f: _ []) =
+        let r = random 1.0
+    
+        let rec find =
+            function
+            | 0 -> fst f.[0]
+            | n when r < snd f.[n] -> fst f.[n]
+            | n -> find (n - 1)
+    
+        find <| f.Length - 1
+    """
+{ FormatConfig.Default with IndentSize = 2 }
+
+(**
+## <img class="gresearch-recommendation align-middle mb-2" data-bs-toggle="tooltip" data-bs-custom-class="gresearch-tooltip" data-bs-title="If you use one of these you should use all G-Research settings for consistency reasons" data-bs-custom-class="orange-tooltip" src="{{root}}/gresearch.svg" alt="G-Research logo"/> G-Research style
```

Comment:
I wouldn't repeat the icon for the h2 title here. I may have said otherwise but seeing it now, I would drop it.